keyboards/spv1/info.json

```
@@ -0,0 +1,138 @@
+{
+    "keyboard_name": "SPV1",
+    "manufacturer": "Bisoromi",
+    "url": "",
+    "maintainer": "Gondolindrim",
+    "usb": {
+        "vid": "0xAC11",
+        "pid": "0x5356",
+        "device_version": "0.0.1"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "matrix_pins": {
+        "cols": ["A5" ,"A6" ,"C4" ,"C5","C8" ,"C9" ,"D2" ,"C12","C11","C10","A10","B3" ,"B5" ,"B4" ,"B8" ,"B9" ,"C13","C14","C15" ],
+        "rows": ["C3" ,"C2" ,"C1" ,"C0","A15","A7" ,"A0" ]
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "backlight": true
+    },
+    "backlight": {
+        "breathing": true,
```

migrated from `rules.mk`
```suggestion
        "driver": "pwm",
        "breathing": true,
```

---

Done in 23530fcafc